index.js

```diff
@@ -59,6 +59,8 @@ Auth.prototype.getAuthClient = function (callback) {
     authClient.keyFile = keyFile;
     authClient.email = config.email;
     addScope(null, authClient);
+  } else if (config.apiKey) {
+    googleAuth.fromAPIKey(config.apiKey, addScope);
```

Comment:
This is `addScope` below. Will it still have the same arguments and be able to function the same way?

```js
function addScope(err, authClient, projectId) {
  if (err) {
    callback(err);
    return;
  }

  if (authClient.createScopedRequired && authClient.createScopedRequired()) {
    if (!config.scopes || config.scopes.length === 0) {
      var scopeError = new Error('Scopes are required for this request.');
      scopeError.code = 'MISSING_SCOPE';
      callback(scopeError);
      return;
    }
  }

  authClient.scopes = config.scopes;
  self.authClient = authClient;
  self.projectId = projectId || authClient.projectId;

  callback(null, authClient);
}
```